src/articles/articles.service.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { ArticleDto } from './article.dto';
+import { Article } from './article.entity';
+
+@Injectable()
+export class ArticlesService {
+  constructor(
+    @InjectRepository(Article)
+    private articleRepository: Repository<Article>,
+  ) {}
+
+  async createArticle(articleDto: ArticleDto): Promise<Article> {
+    const article = new Article();
+    article.title = articleDto.title;
+    article.text = articleDto.text;
+    return this.articleRepository.save(article);
+  }
+
+  async findById(id: number): Promise<Article> {
+    const article = await this.articleRepository.findOne({id});
+    if (!article) {
+      throw new HttpException({
```

Comment:
Working with HTTP is the responsibility of the controller, not the service. 
Try to think of layers of logic and divide logic regarding those layers.
Could discuss this on the call.
https://ru.wikipedia.org/wiki/%D0%A0%D0%B0%D0%B7%D0%B4%D0%B5%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5_%D0%BE%D1%82%D0%B2%D0%B5%D1%82%D1%81%D1%82%D0%B2%D0%B5%D0%BD%D0%BD%D0%BE%D1%81%D1%82%D0%B8